.github/workflows/main.yml

```diff
@@ -129,29 +123,27 @@ jobs:
           name: "endlesskeyapp.unsigned.appx"
           path: "endlesskeyapp.unsigned.appx"
 
-      - name: Sign artifact for release
-        id: sign_for_release
-        if: startsWith(github.ref, 'refs/tags/') && github.event.inputs.sign_test_build == false
+      - name: Sign artifact for testing
+        id: sign_for_testing
+        if: startsWith(github.ref, 'refs/tags/') == false
         shell: pwsh
         env:
-          SIGNPATH_ORG_ID: ${{ secrets.SIGNPATH_ORGANIZATION_ID }}
-          SIGNPATH_USER_TOKEN: ${{ secrets.SIGNPATH_API_TOKEN }}
+          SIGN_PHRASE: ${{ secrets.SELFSIGN_PHRASE }}
         run: |
-          Set-PSRepository PSGallery -InstallationPolicy Trusted
-          Install-Module -name SignPath
-          Submit-SigningRequest `
-            -CIUserToken $env:SIGNPATH_USER_TOKEN `
-            -OrganizationId $env:SIGNPATH_ORG_ID `
-            -ProjectSlug Endless_Apps `
-            -SigningPolicySlug Endless_Apps_Signing_for_Microsoft_Store `
-            -ArtifactConfigurationSlug Endless_Key_APPX_Package_for_Microsoft_Store `
-            -InputArtifactPath endlesskeyapp.unsigned.appx `
-            -OutputArtifactPath endlesskeyapp.appx `
-            -WaitForCompletion
+          & "${Env:ProgramFiles(x86)}\Windows Kits\10\App Certification Kit\signtool.exe" `
+            sign /fd SHA256 /a /f .buildkite_win/selfsign.pfx /p $env:SIGN_PHRASE endlesskeyapp.unsigned.appx
+          mv endlesskeyapp.unsigned.appx endlesskeyapp.testsigned.appx
 
-      - name: Sign artifact for testing
-        id: sign_for_testing
-        if: github.event.inputs.sign_test_build
+      - name: Upload signed APPX for testing
+        if: steps.sign_for_testing.outcome == 'success'
+        uses: actions/upload-artifact@v2
+        with:
+          name: "endlesskeyapp.testsigned.appx"
+          path: "endlesskeyapp.testsigned.appx"
+
+      - name: Sign artifact for release
```

Comment:
We don't need to sign this for release -- it looks like the Microsoft Store accepts an unsigned package, so there is no need for this step anymore.

Reply:
Sorry, just to be clear -- we are planning to use SignPath in the future to sign the .exe for the Endless Key USB, but that should be handled in a separate ticket, as we need to define a new signing policy and artifact configuration on their platform first.

Reply:
ok